src/masquerade/providers/web_api.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# * coding: utf8 *
+"""
+A module that contains methods for querying the AGRC geocoding service
+"""
+import os
+
+import requests
+from sweeper.address_parser import Address
+
+WEB_API_URL = 'https://api.mapserv.utah.gov/api/v1/geocode'
+
+
+def get_address_candidates(single_line_address, out_spatial_reference, max_locations):
+    """ parses the single line address and passes it to the AGRC geocoding service
+    and then returns the results as an array of candidates
+    """
+
+    try:
+        parsed_address = Address(single_line_address)
+    except Exception:
+        return []
+
+    parameters = {
+        'apiKey': os.getenv('WEB_API_KEY'),
+        'spatialReference': out_spatial_reference,
+        'suggest': max_locations
```

Comment:
**nitpick**: the web api only allows 5 suggestions

Reply:
Oh really? I thought that was just a limit of the dropdown in the docs page. This request seems to return more than five suggestions: http://api.mapserv.utah.gov/api/v1/geocode/123%20s%20main/slc?suggest=10&apiKey=AGRC-Dev

With that being said, I should probably cap this as well, no? Maybe at 50 just like the suggest endpoint?

Reply:
you caught my lie, there is no hard fast cap in the web api. I'll be adding something though. I wonder if https://github.com/agrc/warehouse/blob/master/mapserv/locatorsupport/templates.py#L15 is a limit.

Reply:
I don't think that there is a limit of `10` either since http://api.mapserv.utah.gov/api/v1/geocode/123%20s%20main/slc?suggest=15&apiKey=AGRC-Dev returns 11 candidates. But yes, I can see why you may want to add one in the future. If you remember, let me know when you do and I'll make this project match the limit. For now, I don't see any real problems with 50 as it is now.